tests/unit/customizations/ec2/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Surprising we did not have any unit test for the previous ec2 customizations... At least there were functional tests.